### ISSUE TITLE
Allow user to show two isochrones with different traveltime

### ISSUE THREAD

**Issue by EPajares** [goat-community/goat#1686]
# Goal of this issue
As discussed in a meeting yesterday, we have the issue that it is impossible to show two isochrones with different travel times. As  we have one slider controlling the travel time for both of them.

![image](https://user-images.githubusercontent.com/42171996/200112867-1a6471de-566a-4387-957a-58eb1b599105.png)

I believe we need two sliders, but currently, we are uncertain where to place them. I personally favouring putting them here for each isochrone:
![image](https://user-images.githubusercontent.com/42171996/200112904-0fabb59c-f210-4174-938b-3296a8bd0b3f.png)

Basically, underneath the address.

@AnaSasso: could you maybe design some rough mockups?
@Ebubeker: it would be great if you get involved and help Ana with feedback. And do the implementation. 

# Resources

# Deliverables

# Branch to derive
https://github.com/goat-community/goat/tree/feature/isochrone-improvements
